utils/measure/measure/util/measure_util.py

```diff
@@ -50,8 +147,13 @@ def take_measurement(
             _LOGGER.debug("Taking sample %d", i)
             error = None
             measurement: PowerMeasurementResult | None = None
+
             try:
-                measurement = self.power_meter.get_power()
+                if self.dummy_load_value:
+                    measurement = self.power_meter.get_power(include_voltage=True)
+                else:
+                    measurement = self.power_meter.get_power()
```

Comment:
I agree long ternary as ruff suggested is not more readable.
But we can just simplify like this.

```suggestion
                self.power_meter.get_power(include_voltage=bool(self.dummy_load_value))
```

Reply:
good catch, thanks!